src/code/z_bgcheck.c

```diff
@@ -2913,25 +2913,25 @@ void DynaPoly_ExpandSRT(PlayState* play, DynaCollisionContext* dyna, s32 bgId, s
     }
 }
 
-void func_8003F8EC(PlayState* play, DynaCollisionContext* dyna, Actor* actor) {
+void DynaPoly_ResetInteractFlagsIfBgActor(PlayState* play, DynaCollisionContext* dyna, Actor* actor) {
     DynaPolyActor* dynaActor;
     s32 i;
 
     for (i = 0; i < BG_ACTOR_MAX; i++) {
-        if (dyna->bgActorFlags[i] & 1) {
+        if (dyna->bgActorFlags[i] & BG_DYNA_ACTOR_FLAG_IS_USED) {
             dynaActor = DynaPoly_GetActor(&play->colCtx, i);
             if (dynaActor != NULL && &dynaActor->actor == actor) {
-                func_800434A0((DynaPolyActor*)actor);
+                DynaPoly_ResetInteractFlags((DynaPolyActor*)actor);
                 return;
             }
         }
     }
 }
 
 /**
- * DynaPolyInfo_setup
+ * Original name: "DynaPolyInfo_setup"
```

Comment:
```suggestion
 * "DynaPolyInfo_setup"
```
I think quotations is enough to imply an og name?

Reply:
On the contrary, by itself that's pretty meaningless (especially once lots of names change).

Reply:
agree that it should probably be kept
though, Im kinda thinking that in the case of functions who's original name are from printfs from within the function, its not as important to label like this. The info is right there in the function if you just look down.

it makes sense for original file names and struct members where that info is more hidden

Reply:
I think it would be a good idea to come up with a consistent way to do this that slots into the Doxygen. In MM I've been using `@remark` for this.

Reply:
:eyes: :point_right: #1265 / discord